server/modules/category/repository/categoryRepository.js

```diff
@@ -1,11 +1,12 @@
 import * as R from 'ramda'
 
-import { db } from '@server/db/db'
-import * as DbUtils from '@server/db/dbUtils'
+import * as DB from '../../../db'
+import { db } from '../../../db/db'
+import * as DbUtils from '../../../db/dbUtils'
 
-import * as Category from '@core/survey/category'
-import * as CategoryLevel from '@core/survey/categoryLevel'
-import * as CategoryItem from '@core/survey/categoryItem'
+import * as Category from '../../../../core/survey/category'
+import * as CategoryLevel from '../../../../core/survey/categoryLevel'
+import * as CategoryItem from '../../../../core/survey/categoryItem'
 
```

Comment:
are there any reason to make the path relative?

Reply:
It prepares the process of getting rid of the server side compilation...